schema.sql

```diff
@@ -76,3 +76,17 @@ CREATE TABLE visits (
   FOREIGN KEY (vet_id) REFERENCES vets(id),
   FOREIGN KEY (animals_id) REFERENCES animals(id)
 );
+/*Add email column to owners table*/
+-- ALTER TABLE owners ADD COLUMN email VARCHAR(120);
+
+/*Drop Not Null constraints on owners table full_name & age column*/
+ALTER TABLE owners ALTER COLUMN full_name DROP NOT NULL;
+ALTER TABLE owners ALTER COLUMN age DROP NOT NULL;
+--- 
+
+--- let create the index: 
+ CREATE INDEX idx_animal_id ON visits (animals_id);
+
+ CREATE INDEX idx_visits_id ON visits (vet_id);
```

Comment:
Nice job so far guys😉. for the vet_id, as shown from your query result in your PR, I noticed the execution time (749.125ms) before introducing the explain analyze command was less than 1000ms (1s) and it is clearly stated in the [requirements ](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_performance_audit.md)that the execution time must be greater than 1000ms. kindly read the [instructions](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_performance_audit.md) carefully and make the changes as required😉.
<img width="1007" alt="image" src="https://github.com/leloufadel/clinic-vet/assets/64643995/53209bd6-d552-4480-aa4d-897a6660d7b1">